code/lib/builder-webpack5/package.json

```diff
@@ -77,6 +77,7 @@
     "@storybook/router": "7.1.0-alpha.3",
     "@storybook/store": "7.1.0-alpha.3",
     "@storybook/theming": "7.1.0-alpha.3",
+    "@swc/core": "^1.3.49",
```

Comment:
I wonder whether we should move `@swc/core` and `swc-loader` to `devDependencies` and `peerDependencies`. We don't want to bloat the install size of our packages further if not necessary. That would also require writing a small documentation about the necessity of installing them. In the long term this would of course then apply to the babel dependencies as well. What do you think? On the other hand this would favour package version conflicts. I am not sure :/

Reply:
I mean putting it into the `devDependencies` for internal reasons. Otherwise, Typescript will complain that the type is not available. As I understand yarn, we don't install peerDeps automatically, therefore this step would be required.